### PR TITLE
Improve layout builder block field labels.

### LINKED ISSUE #2246: Improve field labels for LB blocks
# Problem
Field help text is not always as helpful as it could be.

### PR DESCRIPTION
**Pull request by ethanstjohn**
Fixes #2246 

The goal here is to change all of the uninformative, slightly less understandable, and even missing help text for all of the layout builder types.

This will be an ongoing PR until they are all completed.

Input on what is written is greatly appreciated.

### CODE REVIEW

**Comment by joewhitsitt** [comment]
It would be great if @ABoston88 could look over this diff as it pertains to help text changes. https://github.com/uiowa/uiowa/pull/2513/files

**Comment by joewhitsitt** [comment]
pulling down to test the technical aspect

**Comment by joewhitsitt** [comment]
Configuration imported fine. The non-help text changes seem to just get config up to date (removed dependencies, set headline and text format defaults, update allowed bundles (removed v2 bundles, added contact))

**Review by ABoston88 — CHANGES_REQUESTED** [review]
A few small adjustments to reflect functionality updates: 
-uiowa_banner block: Change to "Add an image or color block that can carry..." etc.
-uiowa_event block: extra apostrophe in "that's"
-uiowa_hero block: Omit "a creative way to" and start with "Display..."

And one question for uiowa_people Sort field description: 'Order People content alphabetically by first name, last name – or by last name, first name. People can also be manually ordered once selected for the page.' Is this accurate? Would it be better to omit the second sentence?

Only that last one would give me pause on going live with this, though.

**Comment by joewhitsitt** [comment]
Thanks for reviewing @ABoston88. I will make these changes. Good catch on the people block. No it doesn't work that way. Sticky/Last/First Sticky/First/Last are the only options in this context.

**Comment by joewhitsitt** [comment]
The extra apostrophe is a code escape character. No change needed there.

**Comment by pyrello** [comment]
Pulling down to test

File: config/default/block_content.type.uiowa_articles.yml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 uuid: cbea77a6-0e50-488a-8d6c-c228b11dc1b9
 langcode: en
 status: true
-dependencies:
-  - sitenow_articles
+dependencies: {  }
```
Comment (pyrello) [inline config/default/block_content.type.uiowa_articles.yml:4]: Is this an intentional change?

Comment (joewhitsitt) [reply config/default/block_content.type.uiowa_articles.yml:4]: i think it was an automatic thing.

File: config/default/field.field.block_content.uiowa_collection.field_uiowa_collection_items.yml [hunk 2]
```diff
@@ -25,52 +25,13 @@ settings:
...
-        enabled: false
-      people:
-        weight: 27
-        enabled: false
-      section:
-        weight: 28
-        enabled: false
-      text:
-        weight: 29
-        enabled: false
```
Comment (pyrello) [inline config/default/field.field.block_content.uiowa_collection.field_uiowa_collection_items.yml:66]: I'm not sure what is going on here. @cory-skeers Can you provide any insight?

Comment (joewhitsitt) [reply config/default/field.field.block_content.uiowa_collection.field_uiowa_collection_items.yml:66]: these are v2 bundles. so they don't exist in a v3 export

File: config/default/field.field.block_content.uiowa_card.field_uiowa_card_title.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@ field_name: field_uiowa_card_title
 entity_type: block_content
 bundle: uiowa_card
 label: Title
-description: ''
+description: 'Choose a headline or title text.'
```
Comment (pyrello) [inline config/default/field.field.block_content.uiowa_card.field_uiowa_card_title.yml:17]: Not sure why, but this doesn't seem to be showing up.

File: config/default/block_content.type.uiowa_articles.yml (hunk 1 above)
Comment (pyrello) [reply config/default/block_content.type.uiowa_articles.yml:4]: I'm not sure this matters. If the dependency exists in code but isn't recognizable via config, we could change it to use `enforced`: https://www.drupal.org/node/2404447

File: config/default/field.field.block_content.uiowa_collection.field_uiowa_collection_items.yml (hunk 2 above)
Comment (pyrello) [reply config/default/field.field.block_content.uiowa_collection.field_uiowa_collection_items.yml:66]: I wonder how they ended up in here in the first place.

File: config/default/field.field.block_content.uiowa_card.field_uiowa_card_title.yml (hunk 3 above)
Comment (richardbporter) [reply config/default/field.field.block_content.uiowa_card.field_uiowa_card_title.yml:17]: We needed to render the description in our custom field widget: 77e2723.

**Comment by ethanstjohn** [comment]
Is this good to merge in at this point?

**Comment by joewhitsitt** [comment]
Sorry, conversation happened off the PR while you were gone @ethanstjohn. @pyrello was looking to spend more time with it:

> Mostly, I’d like to be able to take the time to review the help text in context. I think once this PR gets merged we are going to go back to not thinking about the help text and consider it done, so if that’s the case, I want to take bit more time to finesse it

**Comment by ethanstjohn** [comment]
This makes sense! Thank you

**Comment by joewhitsitt** [comment]
Moving back to draft because I keep scanning the PR list wanting to approve it.

**Comment by pyrello** [comment]
Since I have not been able to complete my review of this, I'm going to say let's go ahead with getting it merged and off our project board. Hopefully, we'll be able to revisit it in the future.

**Review by pyrello — APPROVED** [review]
This looks good for now.

**Pull request merged**